woocommerce-services.php

```diff
@@ -246,7 +246,11 @@ static function get_wcs_admin_script_url() {
 		 * @return string
 		 */
 		static function get_wcs_admin_style_url() {
-			return self::get_wc_connect_base_url() . 'woocommerce-services-' . self::get_wcs_version() . '.css';
+			if ( ! defined( 'WOOCOMMERCE_CONNECT_DEV_SERVER_URL' ) ) {
```

Comment:
It turns out the `woocommerce-services-x.xx.x.css` is only used for the production build (since the code splitting and dynamic loading changes were added). This check of `WOOCOMMERCE_CONNECT_DEV_SERVER_URL` consolidates the check in one place so an empty path will be passed to the assets endpoint